.env.example

```diff
@@ -0,0 +1,41 @@
+# Environment variables for the waste-carriers-renewals service.
+#
+# Many of these variables have suitable defaults built-in to the application,
+# but we list them here for completeness and ease-of-editing.
+#
+# The renewals app also uses environment variables from the frontend config.
+
+# Settings for waste-carriers-frontend
+WCRS_FRONTEND_APP_DOMAIN="http://localhost:3000"
+
+# Application settings
+WCRS_REGISTRATION_EXPIRES_AFTER=3
+WCRS_REGISTRATION_RENEWAL_WINDOW=6
+WCRS_RENEWAL_CHARGE=105
+WCRS_TYPE_CHANGE_CHARGE=40
+WCRS_RENEWALS_DEVISE_SECRET_KEY=
```

Comment:
If I've read the intent wrong, let me know. But it looks like rather than adding all the possible env vars at once, you have gone with adding only what is actually required by the renewals service at this time.

If this is the case I'd be consistent with that intent, and remove from the example things not currently required like this, and the google analytics stuff at the bottom.

Reply:
Good point. I've fixed that (and rebased).